pytest.ini

```diff
@@ -3,7 +3,6 @@
 # UserWarning is due to statement: tensorflow.compat.v2.io import gfile
 # DeprecationWarning is due to statement: tensorflow.compat.v2.io import gfile
 # inspect.getargspec() is invoked in tensorboard/backend/event_processing/event_file_loader.py:61
-# jnp.sum([]) in examples/sst2/train.py:L152 should be replaced with an ndarray.
```

Comment:
Is this change related?
(It's fine to keep it, just wanted to make sure it's on purpose)

Reply:
Sorry I think this got added back in one of the PRs which had merge conflicts and I accidentally did a bad merge.